src/components/Projects/ProjectTableHeader/ProjectTableHeader.jsx

```diff
@@ -65,10 +65,23 @@ const ProjectTableHeader = props => {
        </span> 
       </th>
       <th scope="col" id="projects__inv">
-        {INVENTORY}
+        <span className='d-flex justify-content-between'>
+          {INVENTORY}
+          {/* <Button size='sm' className='ml-2' id='SortingByRecentEditedInventory' onClick={props.handleSort}>
```

Comment:
This code will never be executed and can be removed.

Reply:
This is a needed feature in the future and should be kept.